examples/java/spring-boot/src/test/java/uk/gov/api/springboot/controllers/MetadataControllerTest.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.api.springboot.controllers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import java.util.List;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.api.springboot.models.metadata.ApiMetadata;
+import uk.gov.api.springboot.models.metadata.Data;
+import uk.gov.api.springboot.services.MetadataService;
+
+@ExtendWith(MockitoExtension.class)
+class MetadataControllerTest {
+
+  @Nested
+  class RetrieveAll {
+
+    @InjectMocks MetadataController controller;
+
+    @Mock MetadataService service;
```

Comment:
Personally, I'd say flip to Mocks above the subject under test
```suggestion
    @Mock MetadataService service;

    @InjectMocks MetadataController controller;
```